packages/chain-adapters/src/utxo/UtxoBaseAdapter.ts

```diff
@@ -514,7 +518,12 @@ export abstract class UtxoBaseAdapter<T extends UtxoChainId> implements IChainAd
     const account = await this.getAccount(
       input.pubKey ?? (await this.getPublicKey(wallet, accountNumber, accountType)).xpub,
     )
-    const addresses = (account.chainSpecific.addresses ?? []).map(address => address.pubkey)
+    const addresses = (() => {
+      // This means the account isn't an xpub account, pub a pubkey as account i.e Phantom, which doesn't expose an xpub
```

Comment:
unchained shenanigans when querying a pubkey in the `/account` endpoint

Reply:
updated unchained and now things are nice and clean again